g/glew/stone.yaml

```diff
@@ -0,0 +1,45 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : glew
+version     : 2.2.0
+release     : 1
+homepage    : https://github.com/nigels-com/glew
+upstreams   :
+    - https://downloads.sourceforge.net/glew/glew-2.2.0.tgz : d4fc82893cfb00109578d0a1a2337fb8ca335b3ceccf97b97e5cc7f08e4353e1
+summary     : The OpenGL Extension Wrangler Library
+description : |
+    The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library. GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform. OpenGL core and extension functionality is exposed in a single header file.
+license     :
+    - MIT
+    - BSD
+    - GPL
+builddeps   :
+    - binary(cc)
+    - pkgconfig(egl)
+    - pkgconfig(gl)
+    - pkgconfig(glu)
+    - pkgconfig(x11)
+setup       : |
+    %patch %(pkgdir)/glew-install.patch
+    %patch %(pkgdir)/egl+glx.patch
+  
+    sed -i 's|lib64|lib|' config/Makefile.linux
+    sed -i '/^.PHONY: .*\.pc$/d' Makefile
+build       : |
+    %make STRIP= glew.bin
+    mv bin/glewinfo bin/glxewinfo
+    rm glew.pc
+  
+    %make STRIP= SYSTEM=linux-egl glew.lib.shared bin/glewinfo
+    mv bin/glewinfo bin/eglewinfo
+install     : |
+    %make GLEW_DEST="%(installroot)/usr" STRIP= SYSTEM=linux-egl install
+    install -D -m755 -t "%(installroot)/usr/bin" bin/eglewinfo bin/glxewinfo bin/visualinfo
```

Comment:
We have an install_bin macro: https://github.com/serpent-os/tools/blob/8fa3d1fb5427c4376bcb8f78a31e5d908ca5d1ee/boulder/data/macros/actions/misc.yaml#L14

Reply:
would need 3 lines instead of one.